packages/http/src/resources/Hotspots.ts

```diff
@@ -24,6 +24,16 @@ export default class Hotspots {
     return new Hotspot(this.client, { address })
   }
 
+  async search(term:string): Promise<ResourceList<Hotspot>> {
```

Comment:
Is this more like what you were thinking @allenan?

Reply:
this looks good yeah. definitely keeps the branching logic in the `fetchList` function simpler 👍